stairctl/build.gradle

```diff
@@ -1,29 +1,27 @@
-plugins {
+plugins { 
     id 'application'
     id 'stairway.java-conventions'
 
-    id 'io.spring.dependency-management' version '1.0.13.RELEASE'
-    id 'org.springframework.boot' version '2.7.3'
+    id 'io.spring.dependency-management' version '1.1.4'
+    id 'org.springframework.boot' version '3.2.4'
```

Comment:
We previously reverted this upgrade because the Artifactory publish after merge failed.  This shouldn't happen: I was able to run a "dry run" of Artifactory publishing (dummy credentials) that got past the previous failure when generating the artifacts.